inference-engine/src/mkldnn_plugin/utils/node_dumper.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#ifdef CPU_DEBUG_CAPS
+
+#include "node_dumper.h"
+
+#include "mkldnn_node.h"
+#include "utils/blob_dump.h"
+
+#include "ie_common.h"
+#include <array>
+#include <regex>
+#include <sstream>
+#include <string>
+
+using namespace InferenceEngine;
+
+namespace MKLDNNPlugin {
+
+void NodeDumper::setup() {
+    char* dumpDirEnv = getenv("OV_CPU_BLOB_DUMP_DIR");
+    if (dumpDirEnv)
+        dumpDirName = dumpDirEnv;
+
+    char* dumpAsTextEnv = getenv("OV_CPU_BLOB_DUMP_AS_TEXT");
+    if (dumpAsTextEnv)
+        shouldDumpAsText = true;
+
+    char* dumpInternalBlobsEnv = getenv("OV_CPU_BLOB_DUMP_INTERNAL_BLOBS");
+    if (dumpInternalBlobsEnv)
+        shouldDumpInternalBlobs = true;
+
+    char* filter = getenv("OV_CPU_BLOB_DUMP_NODE_EXEC_ID");
+    if (filter)
+        dumpFilters[FILTER::BY_EXEC_ID] = filter;
+
+    filter = getenv("OV_CPU_BLOB_DUMP_NODE_TYPE");
+    if (filter)
+        dumpFilters[FILTER::BY_TYPE] = filter;
+
+    filter = getenv("OV_CPU_BLOB_DUMP_NODE_LAYER_TYPE");
```

Comment:
Not sure this option is actual. I would propose to delete

Reply:
Will be removed

Reply:
Done